cpp/csp/engine/Scheduler.h

```diff
@@ -6,6 +6,7 @@
 #include <cassert>
 #include <functional>
 #include <map>
+#include <unordered_map>
```

Comment:
this is the only change in this file, my editor auto-removes trailing whitespace.

This won't happen once we finish C++ autoformatting.